### DISCUSSION TITLE
SBAS results

### DISCUSSION THREAD

**Discussion by kjz1997** [gmtsar/gmtsar#688] [General]
Hello, professor

I am a little confused about the results of SBAS. The average annual deformation rate of the sampling points P1~P8 in the study area is negative. When I extract the cumulative deformation of these points, I find that the cumulative deformation is all positive. Is there an error in my processing results?
![gmt_vel](https://user-images.githubusercontent.com/89446964/224525159-6e03f06b-3eee-4f0f-8abf-8c8d33a85c2c.png)
[time_series_P4.txt](https://github.com/gmtsar/gmtsar/files/10949996/time_series_P4.txt)
[time_series_P5.txt](https://github.com/gmtsar/gmtsar/files/10949997/time_series_P5.txt)
[time_series_P6.txt](https://github.com/gmtsar/gmtsar/files/10949998/time_series_P6.txt)
[time_series_P7.txt](https://github.com/gmtsar/gmtsar/files/10949999/time_series_P7.txt)
[time_series_P8.txt](https://github.com/gmtsar/gmtsar/files/10950000/time_series_P8.txt)
[time_series_P1.txt](https://github.com/gmtsar/gmtsar/files/10950001/time_series_P1.txt)
[time_series_P2.txt](https://github.com/gmtsar/gmtsar/files/10950002/time_series_P2.txt)
[time_series_P3.txt](https://github.com/gmtsar/gmtsar/files/10950003/time_series_P3.txt)

**Comment by Xiaohua-Eric-Xu** [comment]
One suggestion I would make here is that you do want to merge your data before putting it into sbas. Otherwise, if you solve time-series first and then paste solutions together, it usually does not look very good. The reference point is different (if you selected one). Atmospheric correction algorithm may chose to solve in a different order. 

For interpretation, the signal is composed of many components. You may have residual atmosphere (all scale, red spectrum), localized subsidence or uplift (small scale), tectonic deformation (large scale or intermediate scale), etc. If they are all positive, consider how you generated the time-series (reference point selection, atmospheric correction, GNSS correction), and see whether it makes sense or if you missed anything.
